mypyc/ir/ops.py

```diff
@@ -1347,6 +1347,28 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_binary_int_op(self)
 
 
+class PtrDeref(RegisterOp):
```

Comment:
For consistency with `LoadInt` and `LoadStatic`, what about calling this `LoadMem`? We can then explain this in terms of loading from memory, instead of pointer dereference. Pointer dereference sounds like a C concept, and for a low-level IR it's better to move closer to machine semantics.